service/src/chain_spec.rs

```diff
@@ -464,6 +465,10 @@ fn minterest_genesis(
 			members: whitelist_members,
 			whitelist_mode: false,
 		},
+		chainlink_feed: ChainlinkFeedConfig {
+			pallet_admin: Some(root_key),
+			feed_creators: endowed_accounts,
```

Comment:
Why are the `endowed_accounts` in the `minterest_genesis`, and the `root_key` in the `testnet_genesis`?
`endowed_accounts` - accounts to which the existential balance should be deposited. There are not the same accounts as `feed_creators`. So, suggestion:
```suggestion
			feed_creators: vec![root_key],
```

Reply:
Fixed. Thanks